tests/bsim/bluetooth/mesh/tests_scripts/dfu/dfu_cli_all_targets_lost.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# Copyright 2023 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+source $(dirname "${BASH_SOURCE[0]}")/../../_mesh_test.sh
+
+# The test instance sequence must stay as it is due to addressing scheme
+# `fail-step` informs `dfu_cli_all_targets_lost` instance what targets it tests.
+conf=prj_mesh1d1_conf
+overlay=overlay_pst_conf
+RunTest dfu_all_tgts_lost \
+	dfu_cli_all_targets_lost \
+	dfu_target_fail_on_metadata \
+	dfu_target_fail_on_metadata \
+	dfu_target_fail_on_metadata \
+	-- -argstest fail-step=1 targets=3
+
+conf=prj_mesh1d1_conf
```

Comment:
I think it is worth to split this files into separate file for each test. This one can take long time to run so it is better if we can parallelize them.

Also, please use different test id (the first parameter of `RunTest`) so that test use different log files.